san/transform.py

```diff
@@ -1,9 +1,14 @@
+"""
+In order to have metrics, which require different order, we need to have transform
+ functions, which reorder or make different dictionaries in general.
+"""
 import operator
 from san.pandas_utils import convert_to_datetime_idx_df
 from functools import reduce
 
 QUERY_PATH_MAP = {
-    'eth_top_transactions': ['ethTopTransactions']
+    'eth_top_transactions': ['ethTopTransactions'],
+    'news': []
```

Comment:
why using empty array here - should `news` be in the `QUERY_PATH_MAP` at all ?
I think we should change this because that should be 2 separate conditions:
```python
    if query + '_transform' in globals():
        result = path_to_data(idx, query, data)
        globals()[query + '_transform'](result)
    else:
        result = data['query_' + str(idx)]
``` 
1. if if query + '_transform' in globals()
2. if query in QUERY_PATH_MAP
and we won't need `news` key here

Reply:
I thought that the idea was to use as little conditions as possible, that's why I made the 'news' key in QUERY_PATH_MAP, so I wouldn't have to use another condition.

Reply:
Also the `news_transform` won't be necessary if this reworked :?

Reply:
Why not? I need it in order to reorder the different columns to not be in alphabetical order

Reply:
The other case is that I remove the upgrade to pandas_utils file and just leave the news be as it is, in that case I really won't need an extra function

Reply:
I think condition in the general function will save defining useless paths in the QUERY_PATH_MAP
 
```python
def transform_query_result(idx, query, data):
    """
    If there is a transforming function for this query, then the result is
    passed for it for another transformation
    """
    if query in QUERY_PATH_MAP:
        result = path_to_data(idx, query, data)
    else:
        result = data['query_' + str(idx)]

    if query + '_transform' in globals():
        globals()[query + '_transform'](result)

    return convert_to_datetime_idx_df(result)
```

and we can remove `news_transform` and `news` key from `QUERY_PAT_MAP`.